docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -12,11 +12,14 @@ When a process instance arrives at such a user task, a new job is created. The p
 
 While it may originally seem like the goal of automating a process is to remove humans entirely, efficiently allocating work through user tasks can be even more beneficial.
 
-In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser.
+In this guide, we’ll step through one way to create an automated process utilizing user tasks – all entirely executable in the browser. If you rather prefer a video based learning experience we got you covered as well. [This Camunda Academy course](https://bit.ly/3PJJocB) features the same content which is being discussed in this guide.
```

Comment:
Same comment here on accessibility. I'd rather not include it if not everyone is able to access it.

Reply:
Yes, the Academy video is accessible to everyone. 👍